### PR TITLE
Fix to 6.3.1

### PR DESCRIPTION
**Pull request by CongMa13**
- CK fixed an issue that prevented it from supporting unit tests with beta is not equal to 0. Add these unit tests back.
- Update changelog for rocm release 6.3

### CODE REVIEW

**Review by amd-jnovotny — APPROVED** [review]
Docs changes are okay. Is there anything for the 6.3.1 changelog or just updates to the 6.3.0 log.

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,28 +2,32 @@
...
-* Various documentation formatting updates and fixes
 * Split kernel instances to improve build times
 
+### Resolved issues
+
+* Fixed a bug in randomized tensor input data generation.
+* Fixed the default strides calculation to be in column major order.
+* Fixed a small memory leak by properly destroying HIP event objects in tests.
+* Default strides calculations now follow column-major convention.
+* Various documentation formatting updates and fixes.
```
Comment (cgmillette) [inline CHANGELOG.md:27]: Can remove line 27 - duplicate of 25

**Comment by cgmillette** [comment]
> Docs changes are okay. Is there anything for the 6.3.1 changelog or just updates to the 6.3.0 log.

The fix is already upstream in CK for 6.3.0. Just re-enabling some tests. We can just keep updates to the 6.3.0 log.

File: CHANGELOG.md (hunk 1 above)
Comment (CongMa13) [reply CHANGELOG.md:27]: removed

**Review by cgmillette — APPROVED** [review]
LGTM

**Pull request merged**